NoteFind/Alerts/ErrorAlerts.m

```diff
@@ -0,0 +1,111 @@
+//
+//  ErrorAlerts.m
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import "ErrorAlerts.h"
+
+@implementation ErrorAlerts
+
++ (void)loginFailure:(LoginViewController *)loginVC {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"login failed" message:@"your email or password was entered incorrectly. please try again" preferredStyle:(UIAlertControllerStyleAlert)];
+    
+    UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"ok" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+    
```

Comment:
If you're completion handlers in this file aren't going to do anything, I would just pass `nil` instead of a block object. This is not only more efficient but also easier to read